cdap-docs/reference-manual/source/faq.rst

```diff
@@ -96,7 +96,8 @@ Yes. You can install CDAP on your Hadoop cluster. See :ref:`install`.
 
 .. rubric:: What Hadoop distributions can CDAP run on?
 
-CDAP has been tested on and supports CDH 4.2.x or later, HDP 2.0 or later, and Apache Hadoop/HBase 2.0.2-0.4 and 2.1.0. 
+CDAP has been tested on and supports CDH 4.2.x through 5.3.x, HDP 2.0 through 2.1, 
+Apache Bigtop 0.8.0, and Apache Hadoop 2.0.2, 2.0.4, and 2.1.0. 
```

Comment:
I think we should leave out "Apache Hadoop 2.0.2, 2.0.4, and 2.1.0."  It's not a full distribution, ie no Hbase.